crates/accelerate/src/two_qubit_decompose.rs

```diff
@@ -2445,14 +2445,16 @@ impl RXXEquivalent {
     }
 }
 
+#[derive(Clone, Debug)]
 #[pyclass(module = "qiskit._accelerate.two_qubit_decompose", subclass)]
 pub struct TwoQubitControlledUDecomposer {
     rxx_equivalent_gate: RXXEquivalent,
+    euler_basis: EulerBasis,
     #[pyo3(get)]
     scale: f64,
 }
 
-const DEFAULT_ATOL: f64 = 1e-12;
+pub const DEFAULT_ATOL: f64 = 1e-12;
```

Comment:
I wonder if we could make the tolerance maybe an optional argument for `TwoQubitControlledUDecomposer` with an internal default, I might be wrong but I don't see the point of exposing a default to just use it by default, we might as well use it internally and avoid the users the hassle.

Reply:
Done in [c2a9ad4](https://github.com/Qiskit/qiskit/pull/13568/commits/c2a9ad452984a4880badf746e28a97d66ffd7ea3)